### ISSUE TITLE
When attempting to view cameras, Homebridge throws UnhandledPromiseRejectionWarning error and child bridge restarts

### ISSUE THREAD

**Issue by senseotech** [Brandawg93/homebridge-nest-cam#422]
**Describe the bug**
When attempting to view cameras, Homebridge throws UnhandledPromiseRejectionWarning error and child bridge restarts

**Debug Output**
[homebridge.log.txt](https://github.com/Brandawg93/homebridge-nest-cam/files/6612473/homebridge.log.txt)

**To Reproduce**

1. Open Home app and navigate to room with camera using this plugin
2. Camera thumbnail updates
3. Tap camera to open live feed
4. loading spinner appears on iOS device, homebridge throws error "6/7/2021, 7:21:38 PM] [homebridge-nest-cam] waiting for socket to connect
(node:10070) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_ARG_TYPE]: The "file" argument must be of type string. Received type function ([Function (anonymous)])
    at validateString (internal/validators.js:124:11)" and after several more exceptions, the child bridge restarts itself

**Expected behavior**

Camera feed loads and is viewable

Please answer these questions before submitting this issue:
```
On what device are you running homebridge: Homebridge-Docker on a Synology NAS
On what iOS version are you having issues: 14.6
What camera types are having issues: Nest Cam Outdoor and Nest Hello Doorbell Cam
Does this issue occur consistently or randomly: Consistently
If applicable, what was the last version that worked properly: It was working on this version until a week or two ago 
```

**Comment by Brandawg93** [comment]
The errors suggest that [this line](https://github.com/Brandawg93/homebridge-nest-cam/blob/master/src/streaming-delegate.ts#L373) is missing in code. Have you updated node recently? Or any other environment changes that you could think of that may have changed?

**Comment by senseotech** [comment]
I migrated the container to another NAS a while back and thought everything was working after that, but the more I thought, the more I realized I didn't actually recall checking streaming feeds after the fact.  Since you mentioned it was like something weird with the code, I hopped in the config UI and re-installed the latest version of the plugin, restarted, and everything is working now.  It appears the migration of the container and files didn't get everything.
